packages/node/src/node.ts

```diff
@@ -84,4 +90,17 @@ export class Node extends Base {
     const buffer = Buffer.from(encoded.buffer, encoded.byteOffset, encoded.byteLength);
     return buffer;
   }
+
+  private createAgent(options: ILogtailOptions) {
+    const family = options.useIPv6 ? 6 : 4;
```

Comment:
💄 I would avoid passing properties inside the class

```suggestion
  private createAgent() {
    const family = this._options.useIPv6 ? 6 : 4;
```